client/src/pages/Profile.jsx

```diff
@@ -26,37 +27,55 @@ const Profile = ({ history, updateTheme, theme }) => {
     setAvatar((prevAvatar) => ({ ...prevAvatar, [valueName]: newValue }));
   }
 
-  return (
-    <main id="profile">
-      <Avatar size="256px" infos={avatar} />
+  const createAvatarComponent = () => {
```

Comment:
This function will be regenerated at each component render (while the function is always the same). It would be better to wrap it into [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) or create a subcomponent.

When using `useCallback`, remember to give to the function some parameters :-)